shared/lib_battery.h

```diff
@@ -433,7 +433,7 @@ class lifetime_calendar_t
 {
 public:
 	lifetime_calendar_t(int calendar_choice, util::matrix_t<double> calendar_matrix, double dt_hour, 
-		float q0=1.02, float a=2.66e-3, float b=7280, float c=930);
+		float q0=1.02, float a=2.66e-3, float b=-7280, float c=930);
```

Comment:
why would this become negative?

Reply:
In the UI it's negative, and since lifetime_calendar_t was always called with all the inputs provided from the UI, the incorrectly provided default value (7280) wasn't ever used

Reply:
Not knowing exactly what b is, does it make sense for it to be negative? Just for thoroughness :)

Reply:
The degradation graph produced looks reasonable. The model is based on this paper here, https://ieeexplore.ieee.org/document/7963578, but I don't see the relevant parameter from a quick skim.